WORKSPACE

```diff
@@ -29,3 +30,11 @@ go_repository(
     commit = "58492e2d83c59ed63881311f46ad6251f77dabc3",
     importpath = "sigs.k8s.io/kustomize",
 )
+
+go_repository(
+    name = "com_github_golangci_golangci-lint",
+    build_file_generation = "on",
+    importpath = "github.com/golangci/golangci-lint",
+    strip_prefix = "golangci-lint-4570c043a9b56ccf0372cb6ba7a8b645d92b3357",
+    urls = ["https://github.com/golangci/golangci-lint/archive/4570c043a9b56ccf0372cb6ba7a8b645d92b3357.zip"],
```

Comment:
Wouldn't it be beneficial to lock it to a [v1.13 version of golangci](https://github.com/golangci/golangci-lint/releases/tag/v1.13) instead of commit sha?

Reply:
No reason, I took this from https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/WORKSPACE#L87-L93 and it has been working fine there.

Reply:
I think it can be simplified to:
```
go_repository(
    name = "com_github_golangci_golangci-lint",
    build_file_generation = "on",
    importpath = "github.com/golangci/golangci-lint",
    tag = "v1.13",
)
```
It is a bit easier to read and later it will be quicker to see which linter version is used. I have also tested this and catches a bit more errors due to newer linter (it found a goimport problem in cmd/clusterctl/validation/validate_cluster_api_objects_test.go)

I can propagate it to cluster-api-provider-aws repo if needed.